project/src/main/java/com/google/sps/data/User.java

```diff
@@ -28,9 +30,15 @@
   /** The idap of the user */
   private String userIdap;
 
+  private List<UserCompetitionInfo> allCompetitions = new ArrayList<>();
```

Comment:
What does this list represent? Every competition this specific user is in? In that case maybe userCompetitions could be a better name. Also should it not be initalised to be an attribute of User? There doesn't appear to be an easy way to access this info at present.